verify-signatures.markdown

```diff
@@ -36,5 +44,6 @@ Our current PGP key fingerprint is `C5B7 4989 65EF D1C2 924B  A9D5 39D3 1987 931
 |:-------------|:-------|:----------------|:----------------|
 | 2022-05-11  | [Issue 2040](https://github.com/opensearch-project/opensearch-build/issues/2040){:target="_blank"}  | 2022-05-12 | 2023-05-12 |
 | 2023-05-04  | [Issue 2136](https://github.com/opensearch-project/opensearch-build/issues/2136){:target="_blank"}  | 2023-05-03 | 2024-05-12 |
+| 2023-06-21  | [Issue 3652](https://github.com/opensearch-project/opensearch-build/issues/3652){:target="_blank"}  | 2023-04-13 | 2024-04-16 |
```

Comment:
May need to get revised based on [opensearch-build discussion](https://github.com/opensearch-project/opensearch-build/issues/3652).

Reply:
Issue link changed in https://github.com/opensearch-project/project-website/pull/1730/commits/a1b7a875f958d60018c665ddab57dd9d754ba591.